src/Directory.Build.props

```diff
@@ -7,8 +7,4 @@
     <PackageTags>aspnetcore;cshtml;razor</PackageTags>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(TargetFrameworkIdentifier)'=='.NETFramework' AND '$(OutputType)'=='library'">
-    <PackageReference Include="NETStandard.Library" Version="$(NETStandardLibrary20PackageVersion)" />
```

Comment:
Can you remove NETStandardLibrary20PackageVersion now as well? Or are there other usages of it?

Reply:
There are other usages.